src/localizePropsDecorator.js

```diff
@@ -21,7 +27,9 @@ export default function localizePropsDecorator({ containerNamespace, local }) {
   const decorator = Component => {
     @skinnable(contains(Component))
     @props({
-      ...globalizedLocalTypes,
+      ___local: t.maybe(t.interface({
```

Comment:
if `localizePropsDecorator` is used by `container`, it means that `container` must be connecting and passing `___local`. This is a maybe because.. it could be still empty (no local ever set by anyone)?